content/articles/wand-library-in-python/index.md

```diff
@@ -0,0 +1,151 @@
+- Image scaling and cropping are also supported.
+- The wand library can read and write images in various formats.
+
+We shall now take a look at how to install wand library.
+
+Let us check the following syntax, and in this case, we will use Ubuntu.
+```bash
+pip install Wand
+```
+The python module wand will be installed in the machine as a version of pip and Python.
+
+Again, we must include the ImageMagick dependencies because the wand library is an Imagick API. 
+```Python
+$ sudo apt-get install libmagickwand-dev  
+```
+### Installation Verification
+After successfully installing the wand python module and Imagemagick dependencies in the machine, we will test by importing Wand. By using an editing environment of your choice, create a Python file and save it with a `.py` extension, e.g., test.py, and then type `import wand` in the created file, and if wand was successfully installed, it would automatically pop up otherwise, wand was not installed.
+```Python
+import wand
+```
+### How can we read an image in Python Wand?
+By utilizing the Wand's picture library's Image module, one will import image files. After that, We will use numerous properties such as height, width, and so to interpret the image. 
+
+Now let us look at a complete example of checking the width and height of an image by following the above three steps.
+```Python
+    # We will start by importing the image library from the Wand.
+from Wand.image import image as wandImage
+   # The second step is to import the image using its correct pathname.
+with wandImage(filename='/wand-library-in-python/python.png') as img:
+       # The third step is to print the image's height and width.
+    print('The Height of the image is:', img.height)
+    print('The width of the image is:', img.width)
+```
+- The first step is to import the image library from the Wand.
+-  The second step in this process is to import an image that you want to get its details.
+- However, to ensure that the image details will be output successfully, the correct pathname should be input and in the correct format. Furthermore, this mainly applies to an image located within several folders. The following format should be used when encountering such an image.
+-  The third step is now to print the details of the image, i.e., the width and the height.
+
+The input image is:
+
+![python](/engineering-education/wand-library-in-python/python.png)
+
+The output is as follows.
+```bash
+The height of the image is: 203
+The width of the image is: 601
+```
+### Use Wand package in Python to blur an image
+We can use several ways in Python to blur an image. For example, one can decide to blur an image using the pillow package, another one can abandon the pillow package and use the OpenCV library, and all will perform the same job. However, in our case, we will use the Wand library to blur the same image, and the following are the steps that we shall use.
```

Comment:
Can you add a new section that compares wand, pillow and openCv? It would really be helpful